### PR TITLE
docs-fr | reference | glossary | controller.md

### PR DESCRIPTION
**Pull request by ryanzidago**

### CODE REVIEW

File: content/fr/docs/reference/glossary/controller.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+aka:
+tags:
+- architecture
+- fundamental
+---
+ Boucle de contrôle surveillant l'état partagé du cluster à travers l'{{< glossary_tooltip text="apiserver" term_id="kube-apiserver" >}} et effectuant des changements en essayant de déplacer l'état actuel vers l'état désiré.
+
+<!--more-->
+
+Parmis les contrôleurs livrés aujourd'hui avec Kubernetes se trouvent le contrôleur de réplication, le contrôleur d'endpoints, de namespace et de serviceacounts.
```
Comment (sftim) [inline content/fr/docs/reference/glossary/controller.md]:
```suggestion
Parmis les contrôleurs livrés aujourd'hui avec Kubernetes se trouvent le contrôleur de réplication, le contrôleur d'endpoints, de namespace et de serviceaccounts.
```
(typo)

Comment (ryanzidago) [reply content/fr/docs/reference/glossary/controller.md]: Fixed it. Thanks!

**Pull request merged**